app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/player/ExclusivePlayer.kt

```diff
@@ -72,10 +79,10 @@ fun SwitchComponent(checked: MutableState<Boolean>) {
       onCheckedChange = { checked.value = it },
       colors =
           SwitchColors(
-              checkedThumbColor = Color.White,
+              checkedThumbColor = pink,
```

Comment:
Those colors maybe should be in the ressource folder, so we can decided the theme like that